### PR TITLE
Update README.md with the New logo

### PR DESCRIPTION
**Pull request by oriollpz**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1 align="center">Angular - The modern web developer's platform</h1>
 
 <p align="center">
-  <img src="aio/src/assets/images/logos/angular/angular.png" alt="angular-logo" width="120px" height="120px"/>
+  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f7/Angular_gradient.png/1200px-Angular_gradient.png" alt="angular-logo" width="120px" height="120px"/>
```
Comment (dimitri-bret) [inline README.md:4]: maybe it should be a good idea to take the asset from the aio folder?

Comment (billyjov) [reply README.md:4]: @dimitri-bret I think it's a good idea to directly include the logo into the assets folder. 
All logos are available under presskit https://angular.dev/press-kit#

Comment (oriollpz) [reply README.md:4]: So far they have not included the logos in the aio folder.

Comment (nektobit) [reply README.md:4]: > So far they have not included the logos in the aio folder.

Hi

Perhaps you should update your pr as follows:
1. Download the logo from the press kit
2. Replace the logo in the folder aio/src/assets/images/logos/angular/angular.png
3. Provide a link to the image inside the aio folder

Comment (JeanMeche) [reply README.md:4]: The logo is used by the AIO docs, It should be not replaced. Both logo should live alongside in the same asset directory.

**Review by JeanMeche — CHANGES_REQUESTED** [review]
Please add the new logo from the presskit to the assets.

**Pull request closed without merging**